Source/options.js

```diff
@@ -512,6 +512,16 @@ $(function() {
     save('ngramMax', 'Maximum word sequence length saved');
   }
 
+  function validate_yandexTranslatorApiKey(){
+    var yandexKey = e("yandexTranslatorApiKey").value
+    if (yandexKey.match(/Unblock|Block/)){
+      alert("Please enter valid Yandex key.\nRemove Block key / Unblock key from the end of the key")
```

Comment:
I agree.

Also, I suspect that some users will be confused by the sentence "Remove Block key / Unblock key from the end of the key". Maybe it is better to replace it by "Remove 'Block' word from the end of the key".   A separate match check for "Unblock" could trigger the message "Remove 'Unblock'  word from the end of the key".

I am also wondering if we should not simply remove the "Block" or "Unblock" word automatically and silently for the user, if the user accidentally copy-pastes it together with the key..